### PR TITLE
Feature/pda 148

### PR DESCRIPTION
**Pull request by gReis89**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

File: widgets/src/components/TaskDetails/Overview/OverviewHeader.js [hunk 1]
```diff
@@ -1,21 +1,34 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import Typography from '@material-ui/core/Typography';
 import Box from '@material-ui/core/Box';
+import Link from '@material-ui/core/Link';
+import Typography from '@material-ui/core/Typography';
 
-import OverviewNavigation from 'components/TaskDetails/Overview/OverviewNavigation';
+// import OverviewNavigation from 'components/TaskDetails/Overview/OverviewNavigation';
```
Comment (rimselis) [inline widgets/src/components/TaskDetails/Overview/OverviewHeader.js:7]: will be needed?

**Review by rimselis — DISMISSED** [review]

File: widgets/src/components/TaskDetails/Overview/OverviewHeader.js (hunk 1 above)
Comment (gReis89) [reply widgets/src/components/TaskDetails/Overview/OverviewHeader.js:7]: it will be reworked in the future, so I didn't removed completely

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

File: widgets/src/components/TaskList/TaskListContainer.js [hunk 2]
```diff
@@ -224,15 +260,34 @@ class TaskList extends React.Component {
...
+                  <SearchInput value={filter} onChange={this.handleChangeFilter} />
+                </div>
+              </Toolbar>
+              <Tabs
+                indicatorColor="primary"
+                textColor="primary"
+                onChange={this.handleChangeTab}
+                value={activeTab}
+              >
+                <Tab label="MY WORK" disabled />
```
Comment (rimselis) [inline widgets/src/components/TaskList/TaskListContainer.js]: maybe worth moving to i18next.t()?

Comment (gReis89) [reply widgets/src/components/TaskList/TaskListContainer.js]: done!

**Comment by entando-service** [comment]
All is well

**Review by rimselis — APPROVED** [review]

**Pull request merged**